ocaml/xapi/xapi_globs.ml

```diff
@@ -1631,6 +1635,17 @@ let other_options =
     , (fun () -> string_of_int !external_authentication_cache_size)
     , "Specify the maximum capacity of the external authentication cache"
     )
+  ; ( "threshold_last_active"
+    , Arg.Int (fun t -> threshold_last_active := Ptime.Span.of_int_s t)
+    , (fun () ->
+        match !threshold_last_active |> Ptime.Span.to_int_s with
+        | None ->
+            "Cannot be represented as int"
```

Comment:
Should this not be an error and never happen?

Reply:
Here https://github.com/xapi-project/xen-api/pull/5976/commits/423c3bf96d3d84e25ce7c6461b075fa8352e0afd.